test/integration/targets/azure_rm_virtualmachine/tasks/boot_diagnostics.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Create a virtual machine with boot diagnostics enabled (default storage account)
```

Comment:
can you combine the boot_diagnostics argument test into original virtualmachine.yml, to reduce vm resources created, and test execution time?

Reply:
I will move as much as I can in there to avoid the additional VM creation.  Probably all of it.

Reply:
just a reminder, besides the test change, there's some other comments